utils/verdaccio-starter/project.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This entire project is copy and pasted from the CLI, but I removed most of the code in `index.ts`.